bluemira/builders/tools.py

```diff
@@ -342,3 +347,82 @@ def make_circular_xy_ring(r_inner, r_outer):
     inner = make_circle(r_inner, center=centre, axis=axis)
     outer = make_circle(r_outer, center=centre, axis=axis)
     return BluemiraFace([outer, inner])
+
+
+def build_sectioned_xy(
+    face: BluemiraFace, plot_colour: Tuple[float]
+) -> List[PhysicalComponent]:
+    """
+    Build the x-y components of sectioned component
+
+    Parameters
+    ----------
+    face: BluemiraFace
+        xz face to build xy component
+    plot_colour: Tuple[float]
+        colour tuple for component
+
+    """
+    xy_plane = BluemiraPlane.from_3_points([0, 0, 0], [1, 0, 0], [1, 1, 0])
+
+    r_ib_out, r_ob_out = find_xy_plane_radii(face.boundary[0], xy_plane)
+    r_ib_in, r_ob_in = find_xy_plane_radii(face.boundary[1], xy_plane)
+
+    sections = []
+    for name, r_in, r_out in [
+        ["inboard", r_ib_in, r_ib_out],
+        ["outboard", r_ob_in, r_ob_out],
+    ]:
+        board = make_circular_xy_ring(r_in, r_out)
+        section = PhysicalComponent(name, board)
+        section.plot_options.face_options["color"] = plot_colour
+        sections.append(section)
+
+    return sections
+
+
+def build_sectioned_xyz(
+    face: BluemiraFace,
+    name: str,
+    n_TF: int,
+    plot_colour: Tuple[float],
+    degree: float = 360,
+    working: bool = False,
+) -> List[PhysicalComponent]:
+    """
+    Build the x-y-z components of sectioned component
+
+    Parameters
+    ----------
+    face: BluemiraFace
+        xz face to build xyz component
+    name: str
+        PhysicalComponent name
+    n_TF: int
+        number of TF coils
+    plot_colour: Tuple[float]
+        colour tuple for component
+    degree: float
+        angle to sweep through
+    working: bool
+        Switch off sectioning (#1319 Topology issue)
```

Comment:
Is there a name a little more descriptive we could use? Maybe `disable_sectioning`, or something? I think it probably warrants a little more explanation in the docstring as well, as this will be a public facing (and very useful) function.

Reply:
Done